dependency-check-maven-config.xml

```diff
@@ -47,9 +46,6 @@
         <cve>CVE-2018-19362</cve>
         <cve>CVE-2018-19361</cve>
         <cve>CVE-2018-19360</cve>
-        <cve>CVE-2019-16942</cve>
-        <cve>CVE-2019-16943</cve>
-        <cve>CVE-2019-17267</cve>
```

Comment:
❗️All of these CVEs apply to versions before 2.9.10. This suppress block should be removed. If any of these CVEs are still being found after this block is removed, that means that there are old versions of jackson still being pulled in and the CVEs need to be addressed.

Additionally, [CVE-2019-16942](https://nvd.nist.gov/vuln/detail/CVE-2019-16942), [CVE-2019-16943](https://nvd.nist.gov/vuln/detail/CVE-2019-16943), [CVE-2019-17531](https://nvd.nist.gov/vuln/detail/CVE-2019-17531) apply for jackson 2.9.10. I think these will need to be addressed.

Reply:
Done